src/View/View.php

```diff
@@ -111,22 +111,22 @@ class View implements EventDispatcherInterface
     public $helpers = [];
 
     /**
-     * The name of the views subfolder containing views for this View.
+     * The name of the subfolder containing templates for this View.
      *
      * @var string
      */
-    public $viewPath = null;
+    public $templatePath = null;
```

Comment:
This might cause compatibility issues. Public properties are part of the public interface.